Code/Conviva/Source/Events/Observers/AppEventForwarder.swift

```diff
@@ -11,10 +11,12 @@ import AVFoundation
 
 fileprivate let willEnterForeground = UIApplication.willEnterForegroundNotification
 fileprivate let didEnterBackground = UIApplication.didEnterBackgroundNotification
+fileprivate let oldAccessLogEntry = NSNotification.Name.AVPlayerItemNewAccessLogEntry
```

Comment:
Can we eliminate `oldAccessLogEntry` & `newAccessLogEntry` then if they're no longer required?